sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1720,4 +1720,11 @@ object functions {
      UnresolvedFunction(udfName, cols.map(_.expr))
   }
 
+  /**
+   * Parsers the comma separated expression strings into the columns they represent, similar to
+   * DataFrame.selectExpr
+   */
+  def expr(expStrs: String*): Seq[Column] = expStrs.map { expr =>
```

Comment:
We might need to have `@scala.annotation.varargs`.
Also, it'd be better to change `expStrs` into `exprs`  to match the variable name of DataFrame#selectExpr.